mithril-common/src/chain_observer/pallas_observer.rs

```diff
@@ -1,7 +1,11 @@
 use anyhow::{anyhow, Context};
 use async_trait::async_trait;
+use pallas_addresses::Address;
+use pallas_codec::utils::{Bytes, CborWrap, TagWrap};
 use pallas_network::facades::NodeClient;
+use pallas_network::miniprotocols::localstate::queries_v16::{Addr, Addrs, UTxOByAddress, Values};
 use pallas_network::miniprotocols::localstate::{queries_v16, Client};
```

Comment:
This could probably be rewritten as:
```suggestion
use pallas_network::{
    facades::NodeClient,
    miniprotocols::localstate::queries_v16::{self, Addr, Addrs, Client, UTxOByAddress, Values}
};
```